index.bs

```diff
@@ -136,7 +136,7 @@ users need to be able to expect that merely visiting any given link
 won't have implications for the security of their computer,
 or for any essential aspects of their [privacy](https://www.w3.org/2001/tag/doc/ethical-web-principles/#privacy).
 
-For example, an API which allows any website to
+For example, an API that allows any website to
```

Comment:
Technically the which/that thing is a Chicago Manual Of Style thing, not a typo.